maquetado/scss/styles.scss

```diff
@@ -55,9 +55,15 @@
   background-color: $wild-sand;
   display: flex;
   flex-wrap: wrap;
+  padding: 120px;
   margin: 0 100px;
 }
 
+.books-list-container {
+  justify-content: space-between;
+  background-color: $wild-sand;
```

Comment:
If you want your page to have a background color you can set a body class and set the color there.